code/modules/busy_space/air_traffic.dm

```diff
@@ -109,7 +109,7 @@ var/datum/lore/atc_controller/atc = new/datum/lore/atc_controller
 		if("wrong_lang")
 			//Can't implement this until autosay has language support
 		if("emerg")
-			var/problem = pick("hull breaches on multiple decks","unknown life forms on board","a drive about to go critical","asteroids impacting the hull","a total loss of engine power","people trying to board the ship")
+			var/problem = pick("hull breaches on multiple decks","unknown life forms on board","a drive about to go critical","lost attitude control","asteroids impacting the hull","an engine on fire","a total loss of engine power","a malfunctioning bluespace drive","people trying to board the ship","instrument failure and zero visability")
```

Comment:
altitude, visibility

Reply:
Attitude control is an aeronautics thing... But good catch on the visibility.